src/extensions/http_api/client.js

```diff
@@ -218,19 +218,25 @@ class HttpApiClient {
      * @param {string} [baseUrl] - The base url
      */
     makeRequest(method, path, baseUrl) {
+        let pathWithoutHost
+        try {
+            pathWithoutHost = path.replace(new URL(path).origin, '')
+        } catch (e) {
+            pathWithoutHost = path
+        }
+        const fullUri = `${baseUrl}${pathWithoutHost}`
```

Comment:
To be precise, that would be an URL, URI is less precise.
```suggestion
        const fullUrl = `${baseUrl}${path}`
```